src/main/resources/org/rust/ide/liveTemplates/output.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version='1.0' encoding='UTF-8'?>
+
+<templateSet group="Rust">
+    <template name="p" description="println!" toReformat="true" toShortenFQNames="true"
+              value="println!(&quot;$END$&quot;);">
+        <context>
+            <option name="RUST_STATEMENT" value="true"/>
+        </context>
+    </template>
+
+    <template name="pr" description="print!" toReformat="true" toShortenFQNames="true"
+              value="print!(&quot;$END$&quot;);">
+        <context>
+            <option name="RUST_STATEMENT" value="true"/>
+        </context>
+    </template>
+
+    <template name="pd" description="debug println!" toReformat="true" toShortenFQNames="true"
+              value="println!(&quot;$FORMAT$&quot;, $END$);">
+        <variable name="FORMAT" expression="" defaultValue="&quot;{:?}&quot;" alwaysStopAt="true"/>
```

Comment:
Why stop a format string? We have `p`, `pd` and `ppd` to avoid this. 

Sidenote: it would be cool to implement "echoing" behaviour of Kotlin's `soutv` template.